Makefile

```diff
@@ -1,11 +1,12 @@
-.PHONY: compile clean all distclean test debug sdist clean-libuv sdist-upload sdist-libuv
+.PHONY: compile clean all distclean test debug sdist clean-libuv \
+	sdist-upload sdist-libuv docs
```

Comment:
I'm not sure it's OK to have a line break in `.PHONY`. Just add a second section like this:

```
.PHONY: compile clean all distclean test debug sdist clean-libuv
.PHONY: sdist-upload sdist-libuv docs
```